packages/terra-list/docs/ListItem.md

```diff
@@ -1,6 +1,6 @@
 # List Item
 
-The list item provides a pattern for selection/selectability for li surrounded child content. Whenever, the list item has a prop `isSelectable=true` then `aria-label` must be provided to List for accessibility.  The onSelect event is the primary means for interaction, as it yields the associated metaData when valid selection occurs, though individual onClick and onKeyDown events can be applied through the use of customProps if greater granularity is required.
+The list item provides a pattern for selection/selectability for li surrounded child content. Whenever, the list item has a prop `isSelectable=true` then `aria-label` must be provided to List for accessibility.  The onSelect event is the primary means for interaction, as it yields the associated metaData when valid selection occurs, though individual onClick and onKeyDown events can be applied through the use of customProps if greater granularity is required. Both `isSelectable=true` and `isDisabled=true` should be set for a non-selectable item in a list containing a combination of both selectable and non-selectable items, to prevent any accessibility issues.
```

Comment:
> Both `isSelectable=true` and `isDisabled=true` should be set for a non-selectable item in a list containing a combination of both selectable and non-selectable items, to prevent any accessibility issues.

This seems weird. I would expect to only need to supply `isDisabled=true`

Reply:
I take it back, this makes sense because you should only disabled a selectable item

Reply:
@saket2403 I still think that the phrase should be reworded. Why would we want to set `isSelectable=true` for an item that is non-selectable?

Did you mean to say something like this? ⬇️

> Both `isSelectable=true` and `isDisabled=true` should be set to represent the disabled state of a selectable item in a list containing a combination of both selectable and non-selectable items, to prevent any accessibility issues.